docs/literate/src/files/getting_started.jl

```diff
@@ -0,0 +1,250 @@
+# After reading it, you will know how to install Julia and Trixi.jl on your computer,
+# and you will be able to download setup files from our GitHub repository, modify them,
+# and run simulations.
+
+# The contents of this tutorial:
+# - [Julia installation](@ref Julia-installation)
+# - [Trixi.jl installation](@ref Trixi.jl-installation)
+# - [Running a simulation](@ref Running-a-simulation)
+# - [Getting an existing setup file](@ref Getting-an-existing-setup-file)
+# - [Modifying an existing setup](@ref Modifying-an-existing-setup)
+
+
+# ## Julia installation
+
+# Trixi.jl works with the current stable Julia release. More information about Julia support can be
+# found in the [`README.md`](https://github.com/trixi-framework/Trixi.jl#installation) file.
+# A detailed description of the installation process can be found in the
+# [Julia installation instructions](https://julialang.org/downloads/platform/).
+# But you can follow also our short installation guidelines for Windows and Linux below.
+
+
+# ### Windows
+
+# - Download Julia installer for Windows from [https://julialang.org/downloads/](https://julialang.org/downloads/).
+#   Make sure that you choose the right version of the installer (64-bit or 32-bit) according to
+#   your computer.
+# - Open the downloaded installer.
+# - Paste an installation directory path or find it using a file manager (select `Browse`).
+# - Select `Next`.
+# - Check the `Add Julia to PATH` option to add Julia to the environment variables. 
+#   This makes it possible to run Julia in the terminal from any directory by only typing `julia`.
+# - Select `Next`, then Julia will be installed.
+
+# Now you can verify, that Julia is installed:
+# - Press `Win+R` on a keyboard.
+# - Enter `cmd` in opened window.
+# - Enter in a terminal `julia`. 
+
+# Then Julia will be started. To close Julia enter `exit()` or press `Ctrl+d`. 
+
+
+# ### Linux
+
+# - Open a terminal and navigate (using `cd`) to the directory, where you want to store Julia.
+# - To install Julia, get a link to the latest version of Julia from the
+#   [Julia website](https://julialang.org/downloads/), then download an archive file by executing:
+#   ```shell
+#   wget https://julialang-s3.julialang.org/bin/linux/... # your link goes here
+#   ```
+# - Unpack the downloaded file with: 
+#   ```shell
+#   tar xf julia-....tar.gz # your archive filename goes here
+#   ```
+
+# Now you can verify that Julia is installed entering `<Julia directory>/bin/julia`
+# (e.g. `julia-1.8.5/bin/julia`) command in a terminal. `<Julia directory>` is the directory where
+# Julia is installed.
+# Then Julia will be started. To close Julia, enter `exit()` or press `Ctrl+d`.
+
+# Note, that further in the tutorial Julia will be started only typing `julia` in a terminal.
+# To enable that, you have to add
+# [Julia to the PATH](https://julialang.org/downloads/platform/#linux_and_freebsd).
+
+
+# ## Trixi.jl installation
```

Comment:
Thank you! I believe the idea is to combine all the necessary steps for first use in one tutorial so that the newbie doesn't get lost. A pretty big part of the information in these texts has already been mentioned in other tutorials, but it's not always easy to put everything together... But maybe I'm wrong, @sloede, what would you say?

Reply:
Yes, my idea was that this is a one stop shop for new users, without the need to go anywhere else. Repetition is OK here IMHO